components/ui-shell/settings.ts

```diff
@@ -16,6 +17,7 @@ interface NavItem {
   label: string;
   url?: string;
   children?: NavItem[];
+  arrowSvg?: TemplateResult | undefined;
```

Comment:
In this case the icon is an arrow but it could be other thing, so a good practice is to try to not "close" possibilities. In this case you could use:

```suggestion
  icon?: TemplateResult | undefined;
```